<?xml version="1.0" encoding="UTF-8"?>
<launch>
	<!-- Parameters -->
	<arg name="config_filepath" default="$(find ground_remover)/config/config.yaml"/>
	<arg name="algorithm" default="0" />
	<arg name="input_topic" default="/ouster_points_self_filtered"/>
	<arg name="voxel_size" default="0.15"/>
	
	<!-- Launch Node -->
	<node name="ground_remover" pkg="ground_remover"
		type="ground_remover_node" output="screen">
		<param name="config_filepath" value="$(arg config_filepath)"/>
		<param name="algorithm" value="$(arg algorithm)"/>
		<param name="voxel_size" value="$(arg voxel_size)"/>
		<param name="input_topic" value="$(arg input_topic)"/>
	</node>

	<!-- Load Rosparam -->
	<rosparam command="load" file="$(find patchwork)/config/params_ouster128.yaml" />
</launch>
